views/cypress/tests/manage-schema.spec.js

```diff
@@ -90,6 +90,30 @@
                 selectors.editClassUrl
             );
         });
+
+        it('validate restriction - notEmpty', function () {
+            cy.getSettled('span[class="icon-edit"]').last().click();
```

Comment:
Maybe we can group up this in a command?

Reply:
Added